pages/studentPortal/labs/multiplication/game.tsx

```diff
@@ -1,10 +1,14 @@
+import { shuffle } from "lodash";
 import React, { useState } from "react";
+import CalculateWinner from "../../../../components/math/longestStreak/CalculateWinner";
 import MultiplicationBlock, {
   BlockState,
-} from "../../../../components/math/MultiplicationBlock";
-import StatementRow from "../../../../components/math/stories/StatementRow";
+} from "../../../../components/math/longestStreak/MultiplicationBlock";
+import SetRules from "../../../../components/math/longestStreak/SetRules";
 import { Button } from "../../../../components/ui/Button";
+import { getRndInteger } from "../../../api/random";
 
+let initialGameState: GameBlockState[] = [];
 export enum STAGE {
```

Comment:
I see you defined your own stage here already. I think you mistakenly added to the other games stage enum. This enum looks good.